File: languages/c/basics/02_variaveis_tipos.c
/*
Objetivo: revisar tipos básicos e formatação de saída.
Como executar:
  powershell -ExecutionPolicy Bypass -File .\scripts\build-c-module.ps1 -Source .\languages\c\basics\02_variaveis_tipos.c
Conceitos treinados: int, double, char, printf.
Próximo exercício: 03_condicionais_lacos.c
*/

#include <stdio.h>

int main(void) {
    int idade = 31;
    double altura = 1.78;
    char inicial = 'J';

    printf("Idade: %d\n", idade);
    printf("Altura: %.2f\n", altura);
    printf("Inicial: %c\n", inicial);

    return 0;
}

File: languages/fortran/basics/02_variaveis_tipos.f90
! Objetivo: revisar tipos numéricos e lógicos.
! Como executar:
!   powershell -ExecutionPolicy Bypass -File .\scriptsuild-fortran-module.ps1 -Source .\languagesortranasics_variaveis_tipos.f90
! Conceitos treinados: integer, real, logical, character.
! Próximo exercício: 03_condicionais_lacos.f90

program variaveis_tipos
    implicit none
    integer :: idade
    real :: nota
    logical :: aprovado
    character(len=20) :: nome

    idade = 31
    nota = 8.75
    aprovado = .true.
    nome = "Juniormar"

    print *, "Nome:", trim(nome)
    print *, "Idade:", idade
    print *, "Nota:", nota
    print *, "Aprovado:", aprovado
end program variaveis_tipos
